src/vivarium_gates_nutrition_optimization/model_specifications/model_spec.yaml

```diff
@@ -5,20 +5,22 @@ components:
         metrics:
             - DisabilityObserver()
             - DiseaseObserver("maternal_disorders")
+            - DiseaseObserver("maternal_hemorrhage")
     vivarium_gates_nutrition_optimization:
         components:
-            - MaternalMortalityObserver()
             - Pregnancy()
             - MaternalDisorders()
+            - MaternalHemorrhage()
             - MaternalMortality()
+            - ResultsStratifier()
             - PregnancyObserver()
             - BirthRecorder()
-            - ResultsStratifier()
+            - MaternalMortalityObserver()
```

Comment:
Is there a reason for the reordering of components?

Reply:
It helped me keep it straight to roughly group them to (implementation) -> (observers) and then within those groups ordered approximately by model number.